### ISSUE TITLE
Update DPDK version to 18.08?

### ISSUE THREAD

**Issue by guesslin** [intel-go/nff-go#446]
I'm trying to run application on Azure with `uio_hv_generic` driver with `nff-go 0.4.1`.
But always got error `Requested receive port exceeds number of ports which can be used by DPDK (bind to DPDK). (3)` back. Which is returned in https://github.com/intel-go/nff-go/blob/9a5e68c5e09db53d970108ccf081251dfa7e9b7d/flow/flow.go#L494-L497

And then I check with DPDK release note https://doc.dpdk.org/guides/rel_notes/release_18_08.html
```
Added support for Hyper-V netvsc PMD.

The new netvsc poll mode driver provides native support for networking on Hyper-V. See the Netvsc poll mode driver NIC driver guide for more details on this new driver.
```

Is the `uio_hv_generic` downward compatible with `DPDK 18.02`? or which version of nff-go compatible with the driver?

**Comment by gshimansky** [comment]
What does DPDK script `./usertools/dpdk-devbind.py -s` show to you?

We didn't try DPDK 18.08, it is not even released yet. There is a pull request #383 to update to DPDK 18.05, but it wasn't integrated too because there was no demand for it.
I can't say whether you can use a kernel module from 18.08 with DPDK library version 18.02. I think it is better to build NFF-Go with DPDK version 18.08 when it is released.

**Comment by guesslin** [comment]
@gshimansky So DPDK 18.08 is not official released yet.

Here's the result of `./usertools/dpdk-devbind.py -s` after I bound `uio_hv_generic` driver.
```
$ ./usertools/dpdk-devbind.py -s

Network devices using DPDK-compatible driver
============================================
<none>

Network devices using kernel driver
===================================
<none>

Other Network devices
=====================
<none>

Crypto devices using DPDK-compatible driver
===========================================
<none>

Crypto devices using kernel driver
==================================
<none>

Other Crypto devices
====================
<none>

Eventdev devices using DPDK-compatible driver
=============================================
<none>

Eventdev devices using kernel driver
====================================
<none>

Other Eventdev devices
======================
<none>

Mempool devices using DPDK-compatible driver
============================================
<none>

Mempool devices using kernel driver
===================================
<none>

Other Mempool devices
=====================
<none>
```

**Comment by gshimansky** [comment]
I see that there are <none> cards under DPDK-compatible driver. Did you run `dpdk-devbind.py --bind` to bind your network cards to the driver?

BTW, DPDK 18.08 was released today, so this makes it possible to switch to it now.

**Comment by guesslin** [comment]
@gshimansky Thanks, I'll try again with #451 patch.
> Did you run dpdk-devbind.py --bind to bind your network cards to the driver?
No, I bind the driver by our written devbind in go.

**Comment by gshimansky** [comment]
When you get error `Requested receive port exceeds number of ports which can be used by DPDK (bind to DPDK).`, it means that port with requested number could not be found. If you cannot find a port with number 0, it means that there are no ports bound to DPDK driver that I can find. Are you sure that your Go binding code does bind ports to DPDK driver?

**Comment by aregm** [comment]
Implemented in PR #451
